src/mem/Pool.h

```diff
@@ -129,11 +129,11 @@ class MemPools
     /**
      * Sets upper limit in bytes to amount of free ram kept in pools. This is
      * not strict upper limit, but a hint. When MemPools are over this limit,
-     * totally free chunks are immediately considered for release. Otherwise
-     * only chunks that have not been referenced for a long time are checked.
+     * deallocate attempts to release memory to the system instead of pooling.
      */
-    void setIdleLimit(ssize_t new_idle_limit);
-    ssize_t idleLimit() const;
+    void setIdleLimit(ssize_t newLimit) {poolIdleLimit = newLimit;}
+    /// \see void setIdleLimit(ssize_t)
+    ssize_t idleLimit() const {return poolIdleLimit;}
```

Comment:
Just minor polishing suggestions (that already account for the suggested renaming of the data member):

```suggestion
    void setIdleLimit(const ssize_t newLimit) { idleLimit_ = newLimit; }
    /// \copydoc idleLimit_
    ssize_t idleLimit() const { return idleLimit_; }
```